src/sonic-frr/patch/0046-zebra-Modify-show-zebra-dplane-providers-to-give-mor.patch

```diff
@@ -18,44 +19,49 @@ dataplane Outgoing Queue to Zebra: 43
 r1#
 
 Signed-off-by: Donald Sharp <sharpd@nvidia.com>
+---
+ zebra/rib.h          |  1 +
+ zebra/zebra_dplane.c | 18 +++++++++++++-----
+ zebra/zebra_rib.c    | 11 +++++++++++
+ 3 files changed, 25 insertions(+), 5 deletions(-)
 
 diff --git a/zebra/rib.h b/zebra/rib.h
-index 2e62148ea0..b78cd218f6 100644
+index a721f4bac..15f877b66 100644
 --- a/zebra/rib.h
 +++ b/zebra/rib.h
-@@ -630,6 +630,7 @@ static inline struct nexthop_group *rib_get_fib_backup_nhg(
- }
- 
+@@ -628,6 +628,7 @@ extern int rib_add_gr_run(afi_t afi, vrf_id_t vrf_id, uint8_t proto,
```

Comment:
Do we need it as part of this patch?

Reply:
This line is autogenerated. This file is not modified by me in the patch. This is an existing patch, but failed to apply cleanly because of changes in the files.  I manually applied them and created the patch backwards,  Probably the functions must have been moved in the newer files. So, the patch is showing a new line 628.
=======================================================================
Old patch file (0046-zebra-Modify-show-zebra-dplane-providers-to-give-mor.patch)
diff --git a/zebra/rib.h b/zebra/rib.h
index 2e62148ea0..b78cd218f6 100644
--- a/zebra/rib.h
+++ b/zebra/rib.h
@@ -630,6 +630,7 @@ static inline struct nexthop_group *rib_get_fib_backup_nhg(
 }

 extern void zebra_vty_init(void);
+extern uint32_t zebra_rib_dplane_results_count(void);

 extern pid_t pid;
=========================================================================
New patch file(0046-zebra-Modify-show-zebra-dplane-providers-to-give-mor.patch)
diff --git a/zebra/rib.h b/zebra/rib.h
  index a721f4bac..15f877b66 100644
  --- a/zebra/rib.h
  +++ b/zebra/rib.h
  @@ -628,6 +628,7 @@ extern int rib_add_gr_run(afi_t afi, vrf_id_t vrf_id, uint8_t proto,
                            uint8_t instance);
  
   extern void zebra_vty_init(void);
  +extern uint32_t zebra_rib_dplane_results_count(void);
  
   extern pid_t pid;
===========================================================